fleet/lib/fluentbit/fleet.yaml

```diff
@@ -0,0 +1,7 @@
+---
+defaultNamespace: &name fluent-bit
+namespaceLabels:
+  lsst.io/discover: "true"
+  lsst.io/author: "fbegyn"
+labels:
+  bundle: *name
```

Comment:
Please add:

```
helm:
  releaseName: *name
```

As otherwise, changes to the repo structure will change the helm release name.

Reply:
handled in [42debf8](https://github.com/lsst-it/k8s-cookbook/pull/231/commits/42debf8ed65d6a40977706d293c34890c76df0b7)